lib/internal/url.js

```diff
@@ -219,6 +219,14 @@ class URL {
     if (base !== undefined && !(base instanceof URL))
       base = new URL(String(base));
     parse(this, input, base);
+
+    Object.defineProperty(this, Symbol.toStringTag, {
+      value: 'URL',
+      writable: false,
+      enumerable: false,
+      configurable: true
+    });
```

Comment:
The [IDL spec](https://heycam.github.io/webidl/#dfn-class-string) says

> If an object has a class string, then the object must, at the time it is created, have a property whose name is the [@@toStringTag](https://tc39.github.io/ecma262/#sec-well-known-symbols) symbol and whose value is the specified string.

It specifically asked for the *value* of the property to be the class string (`URL` or `URLSearchParams`), which was why I used this form for `URLSearchParams` in the first place.